examples/common.py

```diff
@@ -0,0 +1,135 @@
+import os
```

Comment:
Can you move this common.py file to just the low_level_api folder and remove the __init__.py files?

Reply:
I moved common.py and removed the init.py files